repository_service_tuf_worker/repository.py

```diff
@@ -398,23 +401,66 @@ def bootstrap(
     ) -> Dict[str, Any]:
         """
         Bootstrap the Metadata Repository
-
-        Add the online Keys to the Key Storage backend and the Signed Metadata
-        to the Storage Backend.
         """
-        if payload.get("settings") is None:
-            raise ValueError("No settings in the payload")
-
-        metadata = payload.get("metadata")
-        if metadata is None:
-            raise (ValueError("No metadata in the payload"))
+        tuf_settings = payload.get("settings")
+        if tuf_settings is None:
+            raise KeyError("No 'settings' in the payload")
+
+        root_metadata = payload.get("metadata")
+        if root_metadata is None:
+            raise KeyError("No 'metadata' in the payload")
+
+        # Saves the `Root` metadata in the backend storage service and returns
+        # online public key (it uses the `Timestamp` key as reference)
```

Comment:
Here it's worth pointing out the `online public key` is only one:
```suggestion
        # the online public key (it uses the `Timestamp` key as reference)
```